### PR TITLE
Wrap the section button-group at a higher breakpoint

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Wrap the section button-group at a higher breakpoint as the contents of the navigation does not fit.

## QA
- Hit the demo
- Shrink the viewport to between large and small screen
- See that the header wraps but works nicely

## Details
Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/908

## Screenshots
image

### CODE REVIEW

File: src/pages/ModelsIndex/_models.scss [hunk 1]
```diff
@@ -22,7 +22,7 @@
       pointer-events: none;
     }
 
-    @media (min-width: $breakpoint-medium) {
+    @media (min-width: 876px) {
```
Comment (hatched) [inline src/pages/ModelsIndex/_models.scss:25]: I feel like this needs a comment as to why it's a special value.

**Review by hatched — APPROVED** [review]
Thanks @anthonydillon just one small request!

**Comment by hatched** [comment]
CI failure looks like a network issue with yarn.

File: src/pages/ModelsIndex/_models.scss (hunk 1 above)
Comment (anthonydillon) [reply src/pages/ModelsIndex/_models.scss:26]: No problem, done.

**Comment by anthonydillon** [comment]
Yeah got a few of them today. Thanks for kicking off again.

**Pull request merged**